dotnetv3/AutoScaling/scenarios/AutoScale_Basics/AutoScale_Basics/AutoScaleMethods.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AutoScale_Basics
+{
+    public class AutoScaleMethods
+    {
+        // snippet-start:[AutoScale.dotnetv3.AutoScale_Basics.CreateAutoScalingGroup]
+        /// <summary>
+        /// Creates a new Amazon Auto Scaling group.
+        /// </summary>
+        /// <param name="client">The  initialized Amazon Auto Scaling client object.</param>
+        /// <param name="groupName">The name to use for the new Amazon Auto
+        /// Scaling group.</param>
+        /// <param name="launchTemplateName">The name of the Amazon EC2 launch template
+        /// that will be used to create instances in the group.</param>
+        /// <param name="serviceLinkedRoleARN">The Amazon IAM service-linked role that
```

Comment:
Amazon IAM service-linked role --> AWS Identity and Access Management (IAM) service-linked role

Reply:
fixed